src/helpers/weekDays.ts

```diff
@@ -0,0 +1,9 @@
+export const weekDays: Record<number, string> = {
```

Comment:
I don't think we should specify the exact type here since it's automatically inferred by Typescript based on the contents of this object. Also when we use primitive types like strings, Typescript thinks that the values of this object are strings but in fact they're specific strings (MON, TUE, etc.). So, I wouldn't specify the type here but instead add `as const` to the end of the object to make TS infer the values correctly.

Reply:
Resolved.